packages/lesswrong/components/votes/VoteButton.tsx

```diff
@@ -63,32 +60,35 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 })
 
-const VoteButton = <T extends VoteableTypeClient>({
-  vote, collectionName, document, voteType,
+const VoteButton = ({
+  vote, currentStrength, upOrDown,
   color = "secondary",
   orientation = "up",
   solidArrow,
   theme,
   classes,
 }: {
-  vote: (props: {document: T, voteType: string|null, collectionName: CollectionNameString, currentUser: UsersCurrent})=>void,
-  collectionName: CollectionNameString,
-  document: T,
+  vote: (strength: "big"|"small"|"neutral")=>void,
+  currentStrength: "big"|"small"|"neutral",
   
-  voteType: string,
+  upOrDown: "Upvote"|"Downvote",
   color: "error"|"primary"|"secondary",
   orientation: string,
   solidArrow?: boolean
   // From withTheme. TODO: Hookify this.
   theme?: ThemeType
   classes: ClassesType
 }) => {
-  const currentUser = useCurrentUser();
-  const { openDialog } = useDialog();
-  const { captureEvent } = useTracking();
   const [votingTransition, setVotingTransition] = useState<any>(null);
   const [bigVotingTransition, setBigVotingTransition] = useState(false);
   const [bigVoteCompleted, setBigVoteCompleted] = useState(false);
+  
+  const wrappedVote = (strength: "big"|"small"|"neutral") => {
+    if (currentStrength === "small")
```

Comment:
This line prevents bigVotes, changing them to neutral votes. I think it should be `if (strength === currentStrength)`